src/data_scripts/read_data.py

```diff
@@ -0,0 +1,33 @@
+
+from google.cloud import storage
+import requests
+import pandas as pd
+from io import StringIO
+
+def get_data(url = "https://storage.googleapis.com/cloud-samples-data/ai-platform-unified/datasets/tabular/petfinder-tabular-classification.csv"):
```

Comment:
Also, you seem to have changed the given source of the url `gs://cloud-samples-data/ai-platform-unified/datasets/tabular/petfinder-tabular-classification.csv`
There is a specific lesson in fetching from `gs://` instead of the https equivalence. Try fetching it from the given url.

Reply:
Okay. after I failed to use google cloud bucket I used that. But I will fix it as soon as possible. 
Thank you. I really appreciate the time.